romancal/datamodels/container.py

```diff
@@ -226,6 +235,14 @@ def __iter__(self):
                 model = rdm.open(model, memmap=self._memmap)
             yield model
 
+    def __enter__(self):
+        return self
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        if exc_type is not None:
+            print(f"\nAn exception occurred in ModelContainer: {exc_val}")
+            print(f"\nTraceback:\n{exc_tb}")
```

Comment:
I haven't thought enough about exception handling and context managers.  Do you expect the traceback to not get printed without these three lines?

Reply:
By default (i.e. in the case where we don't add any logic to the method), if an exception occurs inside the context manager, the `.__exit__()` method is called and the traceback will be printed. The implemented code was just a placeholder for handling exceptions; I refactored it to simply `return False` so that, if needed, exceptions can be handled outside the `.__exit__()` method for the moment.